runelite-client/src/main/java/net/runelite/client/plugins/poh/PohPlugin.java

```diff
@@ -24,56 +24,96 @@
 import com.google.common.collect.Sets;
 import com.google.common.eventbus.Subscribe;
 import com.google.inject.Provides;
+import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
 import javax.inject.Inject;
 import lombok.AccessLevel;
 import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Actor;
+import net.runelite.api.AnimationID;
+import net.runelite.api.Client;
 import net.runelite.api.DecorativeObject;
 import net.runelite.api.GameObject;
 import net.runelite.api.GameState;
-import static net.runelite.api.ObjectID.INCENSE_BURNER;
-import static net.runelite.api.ObjectID.INCENSE_BURNER_13209;
-import static net.runelite.api.ObjectID.INCENSE_BURNER_13210;
-import static net.runelite.api.ObjectID.INCENSE_BURNER_13211;
-import static net.runelite.api.ObjectID.INCENSE_BURNER_13212;
-import static net.runelite.api.ObjectID.INCENSE_BURNER_13213;
+import net.runelite.api.ObjectID;
 import net.runelite.api.Tile;
 import net.runelite.api.TileObject;
+import net.runelite.api.events.AnimationChanged;
 import net.runelite.api.events.ConfigChanged;
 import net.runelite.api.events.DecorativeObjectDespawned;
 import net.runelite.api.events.DecorativeObjectSpawned;
 import net.runelite.api.events.GameObjectDespawned;
 import net.runelite.api.events.GameObjectSpawned;
 import net.runelite.api.events.GameStateChanged;
+import net.runelite.api.events.GameTick;
 import net.runelite.client.config.ConfigManager;
+import net.runelite.client.game.HiscoreManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 import net.runelite.client.ui.overlay.OverlayManager;
+import net.runelite.http.api.hiscore.HiscoreClient;
+import net.runelite.http.api.hiscore.HiscoreEndpoint;
+import net.runelite.http.api.hiscore.HiscoreResult;
+import net.runelite.http.api.hiscore.Skill;
 
 @PluginDescriptor(
 	name = "Player-owned House",
 	description = "Show minimap icons and mark unlit/lit burners",
 	tags = {"construction", "poh", "minimap", "overlay"}
 )
+
+@Slf4j
 public class PohPlugin extends Plugin
 {
-	static final Set<Integer> BURNER_UNLIT = Sets.newHashSet(INCENSE_BURNER, INCENSE_BURNER_13210, INCENSE_BURNER_13212);
-	static final Set<Integer> BURNER_LIT = Sets.newHashSet(INCENSE_BURNER_13209, INCENSE_BURNER_13211, INCENSE_BURNER_13213);
+	private static final double ESTIMATED_TICK_LENGTH = 0.6;
+	private final HiscoreClient hiscoreClient = new HiscoreClient();
+
+	@Getter(AccessLevel.PACKAGE)
+	private final Set<Integer> BURNER_UNLIT = Sets.newHashSet(ObjectID.INCENSE_BURNER, ObjectID.INCENSE_BURNER_13210, ObjectID.INCENSE_BURNER_13212);
+
+	@Getter(AccessLevel.PACKAGE)
+	private final Set<Integer> BURNER_LIT = Sets.newHashSet(ObjectID.INCENSE_BURNER_13209, ObjectID.INCENSE_BURNER_13211, ObjectID.INCENSE_BURNER_13213);
+
+	@Getter(AccessLevel.PACKAGE)
+	private double countdownTimer = 130.0; //Minimum amount of seconds a burner will light
```

Comment:
seems unnecessary to initalize this here

Reply:
This would provide the minimum time, if zoning into lit burners. However the PieComponents progress would update accordingly when a player re-lights the burners.

Reply:
Yeah well so set it when needed and not when initializing the plugin